src/web/app/components/sortable-table/sortable-table.component.ts

```diff
@@ -0,0 +1,78 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { SortOrder } from '../../../types/sort-properties';
+
+/**
+ * Column data for sortable table
+ */
+export interface ColumnData<T> {
+  header: string;
+  headerToolTip?: string;
+  sortBy?(item1: T, item2: T): number; // Don't provide this field if you don't want this column to be sortable
```

Comment:
I think it would be better to name as `headerData`? Else it might be misinterpreted as data for the whole column.

Reply:
But the sorting option is for the whole column though.

Reply:
I felt `sortBy` here should be the enum type `SortBy`? 

It should then be passed into tableComparatorService.comapre(), which will handle the comparison logic.

Reply:
Was in the process of changing it. Seems like we had the same idea :)